tests/unit/Exports/OrderExportTest.php

```diff
@@ -605,4 +607,56 @@ function () {
             'storekeeper_id is assigned on wordpress order'
         );
     }
+
+    public function dataProviderStreetNumber()
+    {
+        $streetNumbers = [];
+        $streetNumbers['160'] = [
+            'streetnumber' => '160',
+            'flatnumber' => '',
+        ];
+        $streetNumbers['23'] = [
+            'streetnumber' => '23',
+            'flatnumber' => '',
+        ];
+        $streetNumbers['23-9'] = [
+            'streetnumber' => '23',
+            'flatnumber' => '9',
+        ];
+        $streetNumbers['23-9-A'] = [
+            'streetnumber' => '23',
+            'flatnumber' => '9-A',
+        ];
+        $streetNumbers['146A02'] = [
+            'streetnumber' => '146',
+            'flatnumber' => 'A02',
+        ];
+        $streetNumbers['146A02B'] = [
+            'streetnumber' => '146',
+            'flatnumber' => 'A02B',
+        ];
+        $streetNumbers['20hs'] = [
+            'streetnumber' => '20',
+            'flatnumber' => 'hs',
+        ];
+        $streetNumbers['1011'] = [
+            'streetnumber' => '1011',
+            'flatnumber' => '',
+        ];
```

Comment:
```
        $streetNumbers[' 146 A02B'] = [
            'streetnumber' => '146',
            'flatnumber' => 'A02B',
        ];
        $streetNumbers['146/01'] = [
            'streetnumber' => '146',
            'flatnumber' => '01',
        ];
        $streetNumbers['146 01'] = [
            'streetnumber' => '146',
            'flatnumber' => '01',
        ];
```

It's typed by hand so it can be messy. Add those use cases

Reply:
Added to tests.